### PR TITLE
krankerl: init at 0.13.0

### PR DESCRIPTION
**Pull request by onny**
###### Motivation for this change
Create a package for [krankerl](https://github.com/ChristophWurst/krankerl), a helper utility to develop [Nextcloud](https://nextcloud.com/) apps.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/krankerl/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+, dbus
+, sqlite
+, file
+, gzip
+, makeWrapper
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "krankerl";
+  version = "v0.13.0";
```
Comment (fabaff) [inline pkgs/development/tools/krankerl/default.nix:15]:
```suggestion
  version = "0.13.0";
```

File: pkgs/development/tools/krankerl/default.nix [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  src = fetchgit {
```
Comment (fabaff) [inline pkgs/development/tools/krankerl/default.nix:17]: Use `fetchFromGitHub`.

File: pkgs/development/tools/krankerl/default.nix [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    url = "https://github.com/ChristophWurst/krankerl.git";
+    rev = version;
```
Comment (fabaff) [inline pkgs/development/tools/krankerl/default.nix:19]:
```suggestion
    rev = "v${version}";
```

File: pkgs/development/tools/krankerl/default.nix [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    dbus
+    sqlite
+  ];
+
+  checkInputs = [
+    file
+  ];
+
+  meta = with lib; {
+    description = " A CLI helper to manage, package and publish Nextcloud apps ";
```
Comment (fabaff) [inline pkgs/development/tools/krankerl/default.nix:42]:
```suggestion
    description = "A CLI helper to manage, package and publish Nextcloud apps";
```

File: pkgs/development/tools/krankerl/default.nix [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    homepage = "https://github.com/ChristophWurst/krankerl";
+    license = licenses.gpl3;
```
Comment (fabaff) [inline pkgs/development/tools/krankerl/default.nix:44]: The test apps in the repo are AGPL3+. The `LICENSE` file says GPLv3. Thus `license` should be `gpl3Only`.

```suggestion
    license = licenses.gpl3Only;
```

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 118582` at dcc4753d run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>krankerl</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, please check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/development/tools/krankerl/default.nix:44:5:
```
   |
44 |     license = licenses.gpl3;
   |     ^
```
</li>
  </ul>
</details>

**Comment by onny** [comment]
Thank you @fabaff for the helpful feedback :) Hope all issues are fixed now!

**Pull request merged**